typing/typemod.ml

```diff
@@ -3440,6 +3478,15 @@ let report_error ~loc _env = function
         (Style.as_inline_code longident) lid
         Style.inline_code (Path.name path)
         Style.inline_code (Ident.name id)
+  | With_creates_invalid_aliases(id, path, invalid_path) ->
+      Location.errorf ~loc
+        "@[<v>\
+         @[In this %a constraint, replacing %a by %a would @ \
+         introduce an invalid alias (at %a)@].@]"
```

Comment:
Opening two formatting boxes consecutively with no contents in-between is equivalent to opening only the last box. And in fact no formatting box is needed here. Moreover, it is generally a good idea to have a break hint before arbitrarily large contents. Finally the name of the problematic alias is probably too important to let between parentheses:

```ocaml
Location.errorf ~loc
  "In this %a constraint,@ replacing %a@ by %a@ would@ \
    introduce an invalid alias@ at %a."
```